### PR TITLE
feat: add genesis simulation generation for ics27

### PR DESCRIPTION
**Pull request by colin-axner**
## Description

ref: #1352, #2151

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by seantking — APPROVED** [review]
Nice work!

**Review by damiannolan — APPROVED** [review]
Nice! Is there a specific way to run simulations or is it just included in regular go tests?

File: modules/apps/27-interchain-accounts/simulation/genesis.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	controllertypes "github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/types"
+	genesistypes "github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/genesis/types"
+	hosttypes "github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/host/types"
+	"github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/types"
+)
+
+// RadomEnabled randomized controller or host enabled param with 75% prob of being true.
+func RadomEnabled(r *rand.Rand) bool {
```
[the remark this answers is not in the document]
Comment (colin-axner) [reply modules/apps/27-interchain-accounts/simulation/genesis.go:17]: yes it should be, (copied from transfer)

**Comment by colin-axner** [comment]
> Is there a specific way to run simulations or is it just included in regular go tests?

We have commands in the Makefile, but I think they need some fixing. I tested the 3 prs I opened together on the regen ledger (since one of the issues arose from there). I can try fixing our make file

**Pull request merged**